docs/src/main/asciidoc/spring-web.adoc

```diff
@@ -233,12 +233,30 @@ The following method return types are supported:
 * POJO classes which will be serialized via JSON
 * `org.springframework.http.ResponseEntity`
 
-=== Controller method return types
+=== Controller method parameter types
 
 In addition to the method parameters that can be annotated with the appropriate Spring Web annotations from the previous table,
 `javax.servlet.http.HttpServletRequest` and `javax.servlet.http.HttpServletResponse` are also supported.
 For this to function however, users need to add the `quarkus-undertow` dependency.
 
+=== Exception handler method return types
+
+The following method return types are supported:
+
+* `org.springframework.http.ResponseEntity`
+* `java.util.Map`
+* `void`, but in Quarkus, it must be combined with a `@ResponseStatus` annotation on the method to send an status error to HTTP response. 
```

Comment:
@geoand I think this is not true anymore?

Reply:
Correct, this restriction has been lifted

Reply:
I notice the `void` item has been removed. If the `void` return type is supported, I think we need a `void` in the list. I know the issue is fixed.